src/Portal/Portal.js

```diff
@@ -61,6 +61,8 @@ Portal.propTypes = {
   /**
    * A node, component instance, or function that returns either.
    * The `container` will have the portal children appended to it.
+   * By default, it's using the body of the the top-level document object.
+   * So, it's simply `document.body` most of the time.
```

Comment:
"By default, it uses the body of the the top-level document object, so it's simply"

Reply:
We miss the fact that it's only true 90% of the time.